WORKSPACE

```diff
@@ -220,6 +221,14 @@ pypi_repository(
     version = "3.4.0",
 )
 
+pypi_wheel_repository(
```

Comment:
It greatly simplify the workflow as we can build everything with Bazel, the time to compile the computation module is also reduced from 30 min to a minute. (+ should work on mac)

As we don't have a working version for the c++ serving yet we have no advantage in recompiling Tensorflow.

For now it might be better to focus on creating and training the model and only later on the performance of the serving.

Also note:
* The pip module can use the gpu
* Tensorflow might have a better integration with Bazel soon (no configure or git subrepository needed)